network-upgrades/mainnet-upgrades/byzantium.md

```diff
@@ -0,0 +1,40 @@
+---
+eip: 609
+title: "Hardfork Meta: Byzantium"
+author: Alex Beregszaszi (@axic)
+type: Meta
+status: Final
+created: 2017-04-23
+requires: 100, 140, 196, 197, 198, 211, 214, 607, 649, 658
+link: https://eips.ethereum.org/EIPS/eip-609
+---
+
+## Abstract
+
+This specifies the changes included in the hard fork named Byzantium.
+
+## Specification
+
+- Codename: Byzantium
+- Aliases: Metropolis/Byzantium, Metropolis part 1
+- Activation:
+  - Block >= 4,370,000 on Mainnet
+  - Block >= 1,700,000 on Ropsten testnet
+- Included EIPs:
+  - [EIP-100](./eip-100.md) (Change difficulty adjustment to target mean block time including uncles)
+  - [EIP-140](./eip-140.md) (REVERT instruction in the Ethereum Virtual Machine)
+  - [EIP-196](./eip-196.md) (Precompiled contracts for addition and scalar multiplication on the elliptic curve alt_bn128)
+  - [EIP-197](./eip-197.md) (Precompiled contracts for optimal ate pairing check on the elliptic curve alt_bn128)
+  - [EIP-198](./eip-198.md) (Precompiled contract for bigint modular exponentiation)
+  - [EIP-211](./eip-211.md) (New opcodes: RETURNDATASIZE and RETURNDATACOPY)
+  - [EIP-214](./eip-214.md) (New opcode STATICCALL)
+  - [EIP-649](./eip-649.md) (Difficulty Bomb Delay and Block Reward Reduction)
+  - [EIP-658](./eip-658.md) (Embedding transaction status code in receipts)
```

Comment:
These links will 404, since they are part of a different repository/website.  Throughout these files, the links look well standardized, so you can probably just do a find/replace with a fancy little regex to point them all at `https://eips.ethereum.org/EIPS/$1`